core/domain/collection_domain.py

```diff
@@ -46,11 +46,16 @@
 CMD_ADD_COLLECTION_NODE = 'add_collection_node'
 # This takes an additional 'exploration_id' parameter.
 CMD_DELETE_COLLECTION_NODE = 'delete_collection_node'
+# This takes 2 parameters corresponding to the node indices to be swapped.
+CMD_SWAP_COLLECTION_NODES = 'swap_nodes'
 # This takes additional 'property_name' and 'new_value' parameters and,
 # optionally, 'old_value'.
 CMD_EDIT_COLLECTION_PROPERTY = 'edit_collection_property'
 # This takes additional 'property_name' and 'new_value' parameters and,
 # optionally, 'old_value'.
+# Currently, a node only has exploration_id as its parameter,
```

Comment:
I don't understand this added docstring. There's no reference anywhere around here to exploration_id...

Reply:
I meant that, as skills are removed, the CMD_EDIT_COLLECTION_NODE_PROPERTY is now not used anywhere, so in the future, if more parameters are added to nodes, this command should be used.
Will make it clearer.

Reply:
Done, changed docstring.